desktop-widgets/divelogexportdialog.cpp

```diff
@@ -246,7 +246,7 @@ void DiveLogExportDialog::on_buttonBox_accepted()
 		qPrefDisplay::set_lastDir(fileInfo.dir().path());
 		// the non XSLT exports are called directly above, the XSLT based ons are called here
 		if (!stylesheet.isEmpty()) {
-			QFuture<void> future = exportUsingStyleSheet(filename, ui->exportSelected->isChecked(),
```

Comment:
I reckon it worked because `int` can be cast to `void`. This is perfectly fine (reasonably modern) C++:
```
void test(int num)
{
    return (void)num;
}
```

The documentation of `QFuture` says:
> QFuture\<void\> is specialized to not contain any of the result fetching functions. Any QFuture\<T\> can be assigned or copied into a QFuture\<void\> as well. This is useful if only status or progress information is needed - not the actual result data.

So I wonder if this is a regression in Qt6?

Reply:
seems like it. But on the flip side - it seems that my change isn't 'wrong'...